weeklyChallenges01/ChallengesWithTest01.cs

```diff
@@ -8,37 +8,64 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            if ( num1 == num2)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
         }
 
         public double Subtract(double minuend, double subtrahend)
         {
-            throw new NotImplementedException();
+            double difference = minuend - subtrahend;
+            return difference;
         }
 
         public int Add(int number1, int number2)
         {
-            throw new NotImplementedException();
+            int sum = number1 + number2;
+            return sum;
         }
 
         public int GetSmallestNumber(int number1, int number2)
         {
-            throw new NotImplementedException();
+            if (number1 < number2)
```

Comment:
A way to optimize with a one-liner would be a little syntax sugar with a ternary operator.
`return (number1 < number2) ? number1 : number2; Which is translated as if number1 is smaller than number2, return number1....otherwise return number2.